Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -241,6 +243,61 @@ class Frontend {
       }
     }
 
+    // Load global configuration file
+    // First URLs are created, then they are queried. First match is loaded
+    var configLocations: [URL?] = []
+
+    #if os(Windows)
+      if let localAppData = ProcessInfo.processInfo.environment["LOCALAPPDATA"] {
+        configLocations.append(URL(fileURLWithPath: localAppData))
+      }
+      if let programData = ProcessInfo.processInfo.environment["PROGRAMDATA"] {
+        configLocations.append(URL(fileURLWithPath: programData))
+      }
+    #else
+      if let xdgConfigHome = ProcessInfo.processInfo.environment["XDG_CONFIG_HOME"] {
```

Comment:
I'm a bit unhappy because `$XDG_CONFIG_HOME` is not defined by default on Mac. Do you think we could additionally check for `$HOME/.config/`

Reply:
The XDG spec suggests falling back to $HOME/.config if $XDG_CONFIG_HOME is unset so I think that's a good idea too

Reply:
I don’t see any problem with that and don’t think we need to search `$HOME/.config`. On macOS, you can place your config files in `Application Support`.

> The XDG spec suggests falling back to $HOME/.config if $XDG_CONFIG_HOME is unset so I think that's a good idea too. 

Do you have a reference where the XDG spec says this?

Reply:
https://specifications.freedesktop.org/basedir-spec/latest/#variables

> If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used.

Reply:
Oh, great catch. I think we should follow that advice. @WuerfelDev, could you add that to your PR?